storybooknative/src/types/Style.js

```diff
@@ -10,11 +10,21 @@
 import { type ____DangerouslyImpreciseStyle_Internal } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
 import AnimatedNode from "react-native/Libraries/Animated/src/nodes/AnimatedNode";
 
-type ExactStyles = $Exact<____DangerouslyImpreciseStyle_Internal>;
-type ShapedStyles = $Shape<ExactStyles>;
-
-export type Style = {|
-  ...ShapedStyles,
+// The fontSize style property should be a string for the react-native-web
+type ExactStyles = $Exact<{
+  ...____DangerouslyImpreciseStyle_Internal,
+  +fontSize?: string,
   +borderWidth?: number | AnimatedNode | string,
   +lineHeight?: number | string,
+}>;
+
+export type Style = {|
+  ...ExactStyles,
+  +ios?: ExactStyles,
+  +android?: ExactStyles,
+  +web?: ExactStyles,
+|};
+
+export type OutputStyle = {|
+  ...ExactStyles,
```

Comment:
Maybe this is not necessary to spread `ExactStyles` here and just export it?

Reply:
I tried it, but then there was some a flow error. And with this it works OK